openid-sharedsignals-framework-1_0.md

```diff
@@ -563,6 +572,10 @@ configuration information.
 
 Transmitters have metadata describing their configuration:
 
+version
+
+> OPTIONAL. A string representation of a decimal number including a decimal point and at least one number after the decimal point even if it is 0. This indicates the version of the specification the Transmitter adheres to. The version number is determined according to the Versioning ({{VERSIONING}}) specification. The version number of the current specification is "1.0". If the `version` field is missing in the Transmitter Configuration Metadata, then the version number is assumed to be "0.1".
```

Comment:
- A transmitter may want to support both 0.1 and 1.0 versions. 
- A receiver may want to consume APIs with specific  version

Should the receiver indicate which version it can consume by the headers to the API calls?

Reply:
I think since the API is incompatible between 0.1 and 1.0, a Transmitter wishing to support both will have to provide two different endpoints, so each metadata can still refer to just one version.

Reply:
Should the version be part of the API endpoints? Should the API version part of the API URLs or headers? Without that transmitter would not be able to distinguish receivers. 

I feel, there also should be a notion of which version of the spec the receiver supports to allow version negotiation.